lib/client/node.go

```diff
@@ -56,6 +64,11 @@ func (h *nodes) Update(a *api.Node) (*api.Node, error) {
 
 // Apply updates a node if it exists, or creates a new node if it does not exist.
 func (h *nodes) Apply(a *api.Node) (*api.Node, error) {
+	// When creating a new node, initialize global defaults.
```

Comment:
might be worth adding an extra line in the comment saying "This will not overwrite keys that are already present in the datastore."